### PR TITLE
[com_tags] - feed view  fix Notice:  Undefined property

### PR DESCRIPTION
**Pull request by alikon**
Pull Request for Issue #16951 .

### Summary of Changes
fixed item property

### Testing Instructions
Load tags feed via /index.php?option=com_tags&view=tags&format=feed

### Expected result
feed without Notice

### Actual result
Notices in PHP error log

### CODE REVIEW

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on 4f202b17ed53617d2ea396f15c3537a2a3ed4d67<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16955">issues.joomla.org/tracker/joomla-cms/16955</a>.</sub>

**Comment by brianteeman** [comment]
I have been unable to replicate the reported error but this change looks wrong to me

_Assuming i am reading this correctly before this pr it would display_
**author alias or author**
bit now it displays
**author alias or user id**

and the change to remove the feed email negates the settings in global configuration

**Comment by alikon** [comment]
as it is now in the model  we query only the `#__tags ` table,
so we don't have an author email or author field, 
i'll look if we can join `#__tags` and `#__users`  tables

**Comment by alikon** [comment]
is ready to test now

File: components/com_tags/views/tags/view.feed.php [hunk 1]
```diff
@@ -51,8 +51,8 @@ public function display($tpl = null)
 
 			// Strip HTML from feed item description text
 			$description = $item->description;
-			$author      = $item->created_by_alias ?: $item->author;
-			$date        = ($item->displayDate ? date('r', strtotime($item->displayDate)) : '');
+			$author      = $item->created_by_alias ?: $item->created_user_id;
```
Comment (csthomas) [inline components/com_tags/views/tags/view.feed.php]: Are you sure? `$item->created_user_id`

Comment (alikon) [inline components/com_tags/views/tags/view.feed.php]: fixed
thanks

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 5bc299cf722b02ff784e9e8d8abb568c0177e909<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16955">issues.joomla.org/tracker/joomla-cms/16955</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on 5bc299cf722b02ff784e9e8d8abb568c0177e909<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16955">issues.joomla.org/tracker/joomla-cms/16955</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
RTC after two successful tests.

**Pull request merged**